assets/styles/global/_labeled-input.scss

```diff
@@ -63,7 +63,8 @@
     background-color: transparent;
     outline: 0;
     box-shadow: none;
-    padding: 17px 0 0 0;
+    margin: 17px 0 0 0;
+    padding: 0;
```

Comment:
This padding makes it so the entire box is clickable instead of just the bottom half where the actual input is displayed..